modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader location based on cloud availability zones or regions.
+
+Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning helps decrease networking costs and guarantee lower latency.
+
+xref:develop:consume-data/follower-fetching.adoc[Follower fetching], where you configure consumers to fetch records from the closest replica, can also help reduce overall networking costs. 
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+To enable leader pinning, You must xref:manage:rack-awareness.adoc#configure-rack-awareness[configure rack awareness] on the cluster. If the config_ref:enable_rack_awareness,true,properties/cluster-properties[] cluster configuration property is set to `false`, leader pinning is disabled across the cluster.
+
+== Configure leader pinning
+
+You can use a topic configuration property as well as a cluster configuration property to configure leader pinning.
+
+The following steps assume that you have Redpanda deployed in a multi-AZ or multi-region cluster, and you have configured each broker so that the config_ref:rack,true,properties/broker-properties[] configuration property contains rack IDs corresponding to the AZ IDs.
+
+* Set the topic configuration property `redpanda.leaders.preference`. The property accepts the following string values:
++
+--
+** `none`: Opt out the topic from leader pinning.
+** `racks:<rack1>[,<rack2>,...]`: Specify the preferred location (rack) of all partition leaders for the topic to brokers with rack IDs in this list. The list can contain one or more rack IDs. 
+--
++
+This property inherits the default value from `default_leaders_preference`.
++
+You can list the IDs in any order. Spaces in the list are ignored, i.e. `racks:rack1,rack2` and `racks: rack1,  rack2` are equivalent, but you cannot specify empty rack IDs, for example, `racks: rack1,,rack2`.
+
+* Set the cluster configuration property `default_leaders_preference`. This property sets the default leader pinning configuration for all topics that don’t have `redpanda.leaders.preference` explicitly set. It accepts values in the same format as `redpanda.leaders.preference`. Default: `none` 
++
+This property also affects internal topics such as `__consumer_offsets` and transaction coordinators.
```

Comment:
Actually @kbatuigas , we should probably expand on the why a bit, e.g.

"This property also affects internal topics such as `__consumer_offsets` and transaction coordinators.  This helps to additionally colocate all offset tracking and transaction coordination requests for all clients withing one or more preferred regions/AZs, providing additional E2E latency and networking cost benefits."